react-accordion/my-app/src/components/Tab.tsx

```diff
@@ -0,0 +1,33 @@
+import { PlusButton } from './PlusButton';
+import { Topic } from './Accordion';
+
+type TabProps = {
+  tab: Topic;
+  activeTabId: number;
+  setActiveTabId: (value: number) => void;
+};
+
+export function Tab({ tab, activeTabId, setActiveTabId }: TabProps) {
+  function handleClick() {
+    if (tab.id !== activeTabId) {
+      setActiveTabId(tab.id);
+    } else {
+      setActiveTabId(-1);
+    }
+  }
+  return (
+    <li className="mb-4 tab-container" onClick={handleClick}>
+      <div className="flex items-center justify-between border-b tab-name">
+        <h2 className="pb-2 pl-3 text-2xl text-left">{tab.name}</h2>
+        <PlusButton isActive={tab.id === activeTabId} />
+      </div>
+      {activeTabId === tab.id ? (
+        <div className="mt-2 tab-info">
+          <p className="pl-3 text-left">{tab.info}</p>
+        </div>
+      ) : (
+        ''
+      )}
```

Comment:
This is great example of how we can utilize short circuiting using the `&&` operator. You only care about rendering something if the conditional is true right? If it's false, then we don't want anything to happen.

Here's how we could write this instead
```
{activeTabId === tab.id && (
  <div className="mt-2 tab-info">
    <p className="pl-3 text-left">{tab.info}</p>
  </div>
)}
```

Reply:
Oh ya I forgot you could do that